03week/towersOfHanoi.js

```diff
@@ -19,30 +19,58 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
-  // Your code here
-
+function movePiece(startStack, endStack) {
+  console.log("I am inside move piece", startStack, endStack);
+  var movedPiece = stacks[startStack].pop();
+  return stacks[endStack].push(movedPiece);
 }
 
-function isLegal() {
-  // Your code here
-
+function isLegal(startStack, endStack) {
+  console.log("I am inside the is legal method");
+  var startStackValues = stacks[startStack];
+  var endStackValues = stacks[endStack];
+  var endLength = endStackValues.length;
+  if (endLength == 0) {
```

Comment:
does not handle bad input, such as a non exiting stack.
Does not handle an empty starting stack